python/ray/util/placement_group.py

```diff
@@ -149,6 +149,16 @@ def placement_group(bundles: List[Dict[str, float]],
                 "Bundles cannot be an empty dictionary or "
                 f"resources with only 0 values. Bundles: {bundles}")
 
+        # Remove zero value reousrces from a bundle. This is necessary
+        # because cpp code requires all resources to be > 0.
+        zero_value_resource_key = []
+        for resource_key, value in bundle.items():
+            if value == 0:
+                zero_value_resource_key.append(resource_key)
+
+        for key in zero_value_resource_key:
+            del bundle[key]
```

Comment:
For robustness we do this in C++ instead of Python?

Reply:
ah, that's a good idea. Let me fix that.